### PR TITLE
Fixes One Crew Member Law

### LINKED ISSUE #4096: Only Crew Member Law Can't Be Removed
#### Brief description of the issue
The law circuit that adds "0. Only X is an crew member." to synthetics can not be removed with resets or a law purge.
#### What you expected to happen
The law to be removed with at least a purge.
#### What actually happened
The law is impossible to remove.
#### Steps to reproduce
1.) Get One crew member law board
2.) Have AI or cyborg that can have its laws modified
3.) Modify laws with the board
4.) Attempt to remove the law with purge or reset

#### Additional info:
 - **Server Revision**: master - 2017-10-17 fb158c255bcaf1eed10c64b4e41d504962bc600e 
 - **Anything else you may wish to add** I believe someone else using the one crew member board overrides any current "only crew member" laws. I also believe the only crew member law might override traitor law 0's

### PR DESCRIPTION
**Pull request by MisterLayne**
No clue why this was a zero law in the first place. 

Fixes #4096

### CODE REVIEW

File: code/game/objects/items/weapons/AI_modules.dm [hunk 1]
```diff
@@ -165,12 +165,8 @@ AI MODULES
 
 /obj/item/weapon/aiModule/oneHuman/addAdditionalLaws(var/mob/living/silicon/ai/target, var/mob/sender)
 	var/law = "Only [targetName] is an crew member."
-	if (!target.is_malf_or_traitor()) // Makes sure the AI isn't a traitor before changing their law 0. --NeoFite
-		target << law
-		target.set_zeroth_law(law)
-		lawchanges.Add("The law specified [targetName]")
-	else
-		lawchanges.Add("The law specified [targetName], but the AI's existing law 0 cannot be overriden.")
+	target << law
```
Comment (mistyLuminescence) [inline code/game/objects/items/weapons/AI_modules.dm]: Would `to_chat(target, law)` work here?

Comment (MisterLayne) [reply code/game/objects/items/weapons/AI_modules.dm]: Mhm.

Comment (Belsima) [reply code/game/objects/items/weapons/AI_modules.dm]: pls do then

Comment (MisterLayne) [reply code/game/objects/items/weapons/AI_modules.dm]: I have went through and changed every '<<'. Pushing now.

**Pull request merged**